client/src/partials/journal/modals/trialBalanceMain.body.js

```diff
@@ -2,8 +2,9 @@ angular.module('bhima.controllers')
   .controller('TrialBalanceMainBodyController', TrialBalanceMainBodyController);
 
 TrialBalanceMainBodyController.$inject = [
-  'SessionService', 'TrialBalanceService', 'GridGroupingService', 'GridColumnService',
-  'NotifyService', '$state', '$timeout', 'uiGridConstants',
+  'SessionService', 'TrialBalanceService', 'NotifyService',
+  '$state', 'uiGridConstants', 'uiGridGroupingConstants', '$filter',
```

Comment:
Are grouping constants used any more? I think that functionality was removed with the view as transaction button. If they're no longer used all of these imports can be removed.